src/content/reference/helm-chart-values.mdx

```diff
@@ -510,13 +510,15 @@ The garbage collector service. It automatically scales idle applications to zero
 - `scaleToZeroPeriod`: The duration, in hours, that an application or resource must be idle before the garbage collector scales it to zero. Set to zero to disable.
 - `deleteNamespacePeriod`: The duration, in days, that a namespace must be idle before the garbage collector deletes it. Set to zero to disable.
 - `slackWebhook`: If set, the garbage collector will send a notification when it scales a resource to zero or when it deletes a namespace.
+- `upSessionByLastSyncedFile`: If true, the garbage collector will mark okteto up sessions as ready for GC if no file has been synchronized in 1 hour. Default behaviour is to check for an active connection (regardless of inactivity)
```

Comment:
```suggestion
- `upSessionByLastSyncedFile`: If true, the garbage collector will mark okteto up sessions as inactive for GC if no file has been synchronized. The default behaviour is to check for an active okteto up session (regardless of files being synched or not)
```
@maroshii about the 1h note, is it 1h? or the time configured in the GC? I guess it's the latter, if the GC period id 4h you need to be 4h without synching files to sleep the namespace, right?

Reply:
It's hardcoded to 1h. I don't see why we would need to align this with the gc schedule. It can be maintained independently. We could add an extra config to configure it but it was not in the requirement. @pchico83 let me know if this is a blocker